roles/common_homebrew/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+- name: Check if Homebrew is installed
+  stat:
+    path: /usr/local/bin/brew
+  register: homebrew_presence
+  no_log: true
+
+- name: Install Homebrew
+  command: /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null
+  when: not homebrew_presence.stat.exists
+
+- homebrew_tap:
+    name: neovim/neovim
+    state: present
+
+- homebrew:
+    update_homebrew: true
+    upgrade_all: true
+
+- homebrew:
+    name:
```

Comment:
Instead of keeping a list, you could use a [Brewfile](https://thoughtbot.com/blog/brewfile-a-gemfile-but-for-homebrew). You could occasionally regenerate it with an ansible command

Reply:
I don't think the ansible homebrew module works with brewfiles 😞